dom.bs

```diff
@@ -5089,6 +5095,23 @@ method, when invoked, must run these steps:
  <li><p><var>document</var>'s <a>origin</a> is the <a>origin</a> of the <a>context object</a>'s
  associated <a>document</a>. [[!HTML]]
 
+ <li><p><var>document</var>'s <a for=Document>content type</a> is determined by
+ <var>namespace</var>:
+
+ <dl class=switch>
+  <dt><a>HTML namespace</a>
+  <dd><code>application/xhtml+xml</code>
+
+  <dt><a>SVG namespace</a>
+  <dd><code>image/svg+xml</code>
+
+  <dt><a>MathML namespace</a>
+  <dd><code>application/mathml+xml</code>
```

Comment:
Is MathML something we want to acknowledge in spec-land, or should we just ignore it as a failed experiment?  I guess if no more UAs are realistically planning to implement it, we should probably ignore it, yeah.